pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -453,7 +453,16 @@ def _set_projection(self, view: str) -> Dict:
     def _update_task_incoming(
         self, payload: dict, db_document: DbDocument, start_time: str, **kwargs
     ) -> DbDocument:
-        """Update the task incoming object."""
+        """Update the task incoming object.
+
+        Args:
+            payload: A dictionary containing the payload for the update.
+            db_document: The document in the database to be updated.
+            start_time: The starting time of the incoming tes request.
```

Comment:
Please use `TES` (capitalized) instead of `tes`. Please update this in _all_ docstrings.

Reply:
Done